### DISCUSSION TITLE
Plex hardware decoding (intel)

### DISCUSSION THREAD

**Discussion by LarsNorgaard** [tteck/Proxmox#367] [Help Needed]
I get this error during install:

```
 - Setting Up Hardware Acceleration.../bin/chmod: cannot access '/dev/dri/*': No such file or directory
‼ ERROR 1@75 Unknown failure occurred.
```

**Comment by tteck** [comment]
They're so many Intel CPU generations. Best that I can do is let you read where it all started. https://github.com/tteck/Proxmox/issues/8 I'll make it a discussion.
